agent/util-scripts/pbench-register-tool

```diff
@@ -175,6 +175,16 @@ while true; do
 		shift
 		_do_test_labels=1
 		;;
+	--persistent)
+		shift
+		oldname=${name}
```

Comment:
The name `oldname` ... This is the original value of the `--name`, which you're changing to select a particular class of tool using that name, but which you still want to report at the end. How about `display_name` or `original_name`??